owasp-top10-2017-apps/a7/streaming/deployments/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.7
+    container_name: mysql-a7-streaming
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: streaming
+      MYSQL_DATABASE: streaming
+      MYSQL_USER: streaming
+      MYSQL_PASSWORD: streaming
+      MYSQL_ROOT_HOST: '%'
+    ports:
+      - 3306:3306
```

Comment:
Would you mind changing the ports to:

```yml
ports:
- 3307:3306
```

I couldn't launch the app right off the bat because I already had a service running on `3306`. Changing this to a less common one should do the trick.

I believe that you should only need to change it here, the rest of the code should work just fine with this change since the other apps connect to the database directly through the docker network you set up. 🙂